datacapturing/src/main/java/de/cyface/datacapturing/MessageCodes.java

```diff
@@ -1,11 +1,16 @@
 package de.cyface.datacapturing;
 
+import de.cyface.datacapturing.backend.DataCapturingBackgroundService;
+
 /**
- * This class is a wrapper for all message codes used by the Cyface backend to send inter process communication (IPC)
- * messages.
+ * This class is a wrapper for all message codes used by the Cyface backend to send inner- and inter process
+ * communication (IPC) messages. For safety and efficiency we use LocalBroadcasts for inner process communication
+ * as they are very easy to use and {@link android.os.Message}s for inter-process communication as the
+ * LocalBroadcasts can't be passed between processes.
```

Comment:
Do you think this documentation is appropriately placed here? This gives away quite some implementation details. Javadoc should be more about how to use a class and not about how it is implemented. These facts are of no interest to me, if I need to work with it. 
I guess we should put that under implementation advice in the readme (which probably does not exist yet).